p1pop.html

```diff
@@ -0,0 +1,120 @@
+                    </div>
+                </div>
+                <div class="Skils">
+                    <div class="Tytuł">
+                        <h1>Skils</h1>
+                        </div>
+                        <div class="box1">
+                                <img src="Vector (3).png">
+                            <p style="color:#f5c842 ;">UX/UI</p>
+                            <p>Figma</p>
+                        </div>
+                        <div class="box2">
+                                <img src="bac.png">
+                                <p style="color:#f5c842 ;">Front End</p>
+                                <p>JS,HTML,CSS</p>
+                        </div>
+                        <div class="box3">
+                                <img src="Vector (1).png">
+                                <p style="color:#f5c842 ;">UX/UI</p>
+                                <p>PhP,Node.js</p>
+                        </div>
+                </div>
+                <div class="Portfolio">
+                    <div class="tytul">
+                        <h1>Portfolio</h1>
+                    </div>
+                <div class="tresc">
+                <div class="p1"></div>
+                <div class="p1x"></div>
+                <div class="tekst">
+                    <h1>Design</h1>
+                    <p>Lorem Ipsum is simply dummy text of the printing and typesetting industry.<br> Lorem Ipsum has been the industry's standard dummy text ever since the 1500</p>
+                    <button>View</button>
+                </div>
+                <div class="p2"></div>
+                <div class="p3"></div>
+                <div class="p4"></div>
+                <div class="p5"></div>
+                </div>
+                </div>
+                <div class="Pudelko">
+                    <div class="tyt"><h1>Contact us!</h1></div>
+                    <div class="miejsce">
+                        <div class="p1">
+                            <div class="l1">
+                                <i class="demo-icon icon-search"></i> 
+                                <p> Please describe your question. Send me a message and I will answer you as soon as possible!</p>
+                            </div>
+                            <div class="l2">
+                                <p style="font-weight: bold"> Designer Poland,<br>
+                                     Wrocław</p>
+                                     <p>ul. Tęczowa 12/3 50-000</p>
+                                     <p> +48 123-456-789</p>
+                                     <p style="text-decoration: underline;"> night-coding@dev.com </p>
+                            </div>
+                        </div>
+                        <div class="p2">
+                            <div class="tat"><h1>Let's start!</h1></div>
+                            <label ><input type="text" class="i1" value="Name"></label>
```

Comment:
Te `input`y nie zadziałają, nie mają w ogóle przycisku do wysłania :) Poprawnie powinno to być zrobione jako formularz, zamknięte w tagu `form` 

http://how2html.pl/form-html/